alphastats/loader/generic_loader.py

```diff
@@ -33,7 +34,8 @@ def __init__(
         self.confidence_column = None
         self.software = "Generic"
         self.evidence_df = None
-        self.gene_names_column = None
+        if gene_names_column in self.rawinput.columns.to_list():
+            self.gene_names_column = gene_names_column
```

Comment:
there would be cases where the `gene_names_column` attribute is not defined .. better use `self.gene_names_column = gene_names_column if gene_names_column in self.rawinput.columns.to_list() else None`

Reply:
Already fixed in the next PR.